packages/tools/nodeEditor/src/components/preview/previewManager.ts

```diff
@@ -420,20 +420,21 @@ export class PreviewManager {
             const bakeTransformation = (mesh: Mesh) => {
                 mesh.bakeCurrentTransformIntoVertices();
                 mesh.refreshBoundingInfo();
+                mesh.parent = null;
             };
 
             if (this._globalState.mode === NodeMaterialModes.Material) {
                 switch (this._globalState.previewType) {
                     case PreviewType.Box:
                         SceneLoader.AppendAsync("https://assets.babylonjs.com/meshes/", "roundedCube.glb", this._scene).then(() => {
-                            bakeTransformation(this._scene.meshes[1] as Mesh);
+                            bakeTransformation(this._scene.getMeshByName("__root__")!.getChildMeshes(true)[0] as Mesh);

```

Comment:
Not sure how the preview manager works TBH, but I am assuming there is no chance of loading more than one glTF, right?

Reply:
Yes, there's always a single mesh, because when we change from one mesh to another, we dispose of the previous one. We had a bug here becase at start, there's the hdr skybox which is in the scene, so this._scene.meshes[1] was `__root__` instead of being the right mesh...